src/core/ui/markdown/html/UnifiedConverter.ts

```
@@ -104,41 +53,91 @@ const UnifiedConverter = (): Converter => {
+function alkemioCustomHtmlToMarkdownMdastPlugin() {
+  return (tree: Root) => {
+    // TODO: with the updated version, so we need the em, strong and paragraph changes
+    // Add <br> to empty paragraphs
+    visit(tree, 'paragraph', (node: Element, index: number, parent: Parent) => {
+      if (!node.children || node.children.length === 0) {
+        parent.children[index] = { type: 'html', value: '<br>' } as HTML;
+      }
+    });
+
+    visit(tree, 'em', (node: Element) => {
+      if (node.children && node.children[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)